app/views/admins/weekly_user_stats.mobile.haml

```diff
@@ -0,0 +1,14 @@
+.col-md-9
+  %h2
+    = t(".current_server", date: Time.now.to_date)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.